.buildkite/linters.sh

```diff
@@ -45,12 +45,13 @@ if ! ./tools/scripts/format_website.sh -t &> format_website; then
 fi
 
 pushd vscode_extension
+yarn install
 if ! yarn lint --output-file=yarn_lint; then
```

Comment:
What in the world...did `yarn lint` just silently succeed before on the non-master build?!

Reply:
Yeah. This is not a complete fix, it's just part way there. I'm continuing to investigate, because linters.sh should have failed.